lib/events/volumeListener.js

```diff
@@ -0,0 +1,26 @@
+var SERVICE_ENDPOINT = '/MediaRenderer/RenderingControl/Event';
+
+var initVolumeListener = function(baseListener, callback) {
+
+  var initialized = false;
+
+  baseListener.addHandler(SERVICE_ENDPOINT, function(data) {
+
+    // wait for initial data before callback
+    if (!initialized) {
+      initialized = true;
+      callback(null);
+    }
+
+    baseListener.parser.parseString(data.LastChange, function(err, result) {
+      baseListener.device.state.volume = parseInt(result.Event.InstanceID[0].Volume[0].$.val);
```

Comment:
TypeError: Cannot read property '0' of undefined
    at /Users/stephen/git/node-sonos/lib/events/volumeListener.js:16:84
    at Parser.<anonymous> (/Users/stephen/git/node-sonos/node_modules/xml2js/lib/xml2js.js:255:20)
    at Parser.emit (events.js:95:17)

- When changing EQ settings. Should guard this listener from assumptions